### DISCUSSION TITLE
I would like to know how to download the CMU Mocap dataset and how to visualize the motion capture files contained within it.

### DISCUSSION THREAD

**Discussion by 98brucelee** [lzz19980125/awesome-time-series-segmentation-papers#5] [Q&A] [answered]
I would like to know how to download the CMU Mocap dataset and how to visualize the motion capture files contained within it.

**Comment by lzz19980125** [comment, marked as answer]
I have previously written a blog post about how to use the CMU Mocap dataset. Please check the following link:
https://zhuanlan.zhihu.com/p/448706754.

**Reply by 98brucelee** [reply]
Thank you very much! I will check it right away.
